app/ide-desktop/lib/content/src/index.ts

```diff
@@ -48,35 +48,38 @@ async function fetchTimeout(url: string, timeoutSeconds: number): Promise<unknow
  * download the application config, or one of the compared versions does not match the semver
  * scheme, it returns `true`. */
 async function checkMinSupportedVersion(config: typeof contentConfig.OPTIONS) {
+    let supported = false
     if (config.groups.engine.options.skipMinVersionCheck.value) {
-        return true
-    }
-    try {
-        const appConfig = await fetchTimeout(
-            config.groups.engine.options.configUrl.value,
-            FETCH_TIMEOUT
-        )
-        if (
-            typeof appConfig === 'object' &&
-            // `typeof x === 'object'` narrows to `object | null`, not `object | undefined`
-            // eslint-disable-next-line no-restricted-syntax
-            appConfig !== null &&
-            'minimumSupportedVersion' in appConfig
-        ) {
-            const minSupportedVersion = appConfig.minimumSupportedVersion
-            if (typeof minSupportedVersion === 'string') {
-                const comparator = new semver.Comparator(`>=${minSupportedVersion}`)
-                return comparator.test(contentConfig.VERSION.ide)
+        supported = true
+    } else {
+        try {
+            const appConfig = await fetchTimeout(
+                config.groups.engine.options.configUrl.value,
+                FETCH_TIMEOUT
+            )
+            if (
+                typeof appConfig === 'object' &&
+                // `typeof x === 'object'` narrows to `object | null`, not `object | undefined`
```

Comment:
commented code

Reply:
sorry - it's a comment that's mostly code, but also this comment is no longer relevant after the lints were changed to accept `null`